firefox-ios/firefox-ios-tests/Tests/XCUITests/ExperimentIntegrationTests.swift

```diff
@@ -132,8 +131,9 @@ final class ExperimentIntegrationTests: BaseTestCase {
         let studiesToggle = app.switches.matching(
             NSPredicate(format: "identifier CONTAINS 'settings.studiesToggle'")
         )
-
+        XCTAssertTrue(studiesToggle.element.value as? String == "1")
         studiesToggle.element.tap()
+        XCTAssertTrue(studiesToggle.element.value as? String == "0")
```

Comment:
Could we write the `XCTAssertTrue` as `XCTAssertEqual(studies.Toggle.element.value, 0)`?
https://developer.apple.com/documentation/xctest/xctassertequal

Reply:
Updated